{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -26,23 +26,25 @@ jobs:
   environment-config:
     runs-on: ubuntu-latest
     outputs:
-      stable-python-version: {{ "${{ steps.get-python-version.outputs.python-version }}" }}
+      stable-python-version: {{ "${{ steps.get-compatible-python.outputs.stable-python }}" }}
+      python-matrix: {{ '${{ steps.get-compatible-python.outputs.python-versions }}' }}
     steps:
-      - uses: actions/checkout@v3
-
-      - id: get-python-version
-        uses: MDAnalysis/get-latest-python-version@main
+      - uses: actions/setup-python@v4
         with:
-          last-n-minor-release: 1
+          python-version: "3.11"
```

Comment:
Does this need to be here? Looks like the action installs Python itself anyway, so I'm not sure Python is ever used here?

Reply:
I didn't want to force a Python install in the action itself - that way folks have the ability to modularly re-use the action for whatever crazy purposes they want (i.e. it won't mess with a base Python install in an existing job).

Reply:
Hm, I wonder why the previous action didn't need it -- maybe it just used the base install? Would this action work without the python setup?

Reply:
base install, and not here, I believe tomllib is 3.11+

Reply:
Ahh okay. Just seems like another version of the same problem this action is supposed to fix (manual updating)

Reply:
Just if this version also has to get updated every now and then to stop things from breaking -- but I guess that'll be at a much slower pace than the MDAnalysis versions

Reply:
The alternative here is that we pip install tomli on the base python install (3.10). The issue might be similar though, once the base runner version updates we'll be in need to change this bit of code. As you say though, it's less flaky than the frequency at which I update MDAnalysis compatibility.

Reply:
Yup, let's keep this as-is.